skaffold.yaml

```diff
@@ -51,19 +51,19 @@ deploy:
       - name: network-org-2-peer-1
         chartPath: charts/hlf-k8s
         namespace: org-2
-        valuesFiles: [examples/2-orgs-policy-any/values/org-2-peer-1.yaml]
+        valuesFiles: [examples/2-orgs-policy-any-no-ca/values/org-2-peer-1.yaml]
         imageStrategy:
           helm: {}
         values:
           image: substrafoundation/hlf-k8s
 
     flags:
       install: ["--create-namespace"]
-  # We create the serviceAccounts with the kubectl deployer to ensure the serviceAccounts will stay present for the Helm deletion hooks
-  # By design Skaffold create and delete ressources in a fixed order (Helm > kubectl > kustomize)
-  # Ref: https://github.com/GoogleContainerTools/skaffold/blob/dedd545/pkg/skaffold/runner/new.go#L191-L214 (func getDeployer)
+
+
   kubectl:
     manifests:
-      - examples/serviceAccounts/serviceAccount-orderer.yaml
-      - examples/serviceAccounts/serviceAccount-org-1.yaml
-      - examples/serviceAccounts/serviceAccount-org-2.yaml
```

Comment:
Why do you remove the serviceAccounts part, we don't need it when we use secrets instead of CA enrollment + genesis creation ?

Reply:
Correct, we don't need them with pre-gen secrets. They're only used by the secret deletion hook

Here's the diff between the non-pregen/pregen orderer values

```diff
>   enabled: false   // that's the CA: now disabled
<   serviceAccount:   // no need for a service account
<     name: substra-delete-hook
>   deleteSecrets:     // that's the hook: now disabled
>     enabled: false
>
> genesis:               // the genesis creation: now disabled
>   generate: false
```